### ISSUE TITLE
EDFBrowser 'Save As' Not Working

### ISSUE THREAD

**Issue by davideddie** [Teuniz/EDFbrowser#41]
I'm having issues with the 'Save As' function in EDFBrowser? Love the versatility of this program, but after inverting an upside down ECG waveform using the 'Organize' function, I'm not able to save the modified file. The 'Save As' button is now always greyed out.

I'm sure I used to be able to do this save a modified EDF file. I'm running version 1.62.

David

**Comment by davideddie** [comment]
Hi Teunis,

Thank you for developing this software and thanks so much for getting in touch. I really appreciate your help with this.

I tried the strategy you suggested but unfortunately the waveform is still upside-down in the new EDF file. Any thoughts about what I could be doing wrong?

Here are my steps:

  1.  Open EDF file
  2.  Signal --> Organize --> Invert
  3.  File --> Print --> to EDF

David

**Comment by Teuniz** [comment]
You are right. I just checked and the "print to EDF" function does not take into account the inversion of signals. I'll put it on the todo list.
In the mean time you can use the following workaround: When adding signals using the "Signals" dialog, use the "Subtract" button and then the "Make derivation" button.
It's another way to invert the signals. After that, you can use the "Print to EDF" function.
Let me know how it goes.

**Comment by Teuniz** [comment]
I just fixed it and pushed it to Github. You can do a pull and then re-compile.
It should work now also with inverted signals & print to EDF.

**Comment by davideddie** [comment]
Wow, nice work! I’ll try that now. Thanks so much.

David

**Comment by davideddie** [comment]
Hi,

So sorry, I was unable to compile from GitHub. Unfortunately, I lack the basic expertise necessary to understand the build instructions provided. I’m also not a PC user which complicates things.

Will the modified version of the program eventually be available for download on your website?

Thanks again for all your help with this.

David


—
David Eddie, Ph.D.
Clinical & Research Fellow, Massachusetts General Hospital
Livingston Fellow, Harvard Medical School
Research Scientist, Recovery Research Institute
151 Merrimac St., 6th Floor
Boston, MA 02114
Phone: (617) 643-9194
Fax: (617) 643-1998
Pager #: 27413
[cid:image001.png@01D391EE.D6F37600]

**Comment by Teuniz** [comment]
I hope to release a new windows version this weekend.

**Comment by davideddie** [comment]
Great, thanks a ton.

David

**Comment by Teuniz** [comment]
The new version 1.63 is available.

**Comment by davideddie** [comment]
﻿Fantastic, thanks so much!

David
